commands/createfixpullrequests.go

```diff
@@ -309,6 +366,21 @@ func (cfp *CreateFixPullRequestsCmd) updatePackageToFixedVersion(impactedPackage
 	return packageHandler.UpdateImpactedPackage(impactedPackage, fixVersionInfo)
 }
 
+// getMinimalFixVersion that fixes the current impactedPackage
```

Comment:
// getMinimalFixVersion that fixes the current impactedPackage
-->
// getMinimalFixVersion find the minimal version that fixes the current impactedPackage;